### PR TITLE
support vue3

### PR DESCRIPTION
**Pull request by Chellhooo**
base on origin master([6e53676](https://github.com/jbaysolutions/vue-grid-layout/commit/6e536767937fbde7f1f18435a185e2b9a00c27ef))

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -90,47 +91,68 @@
...
     import GridItem from './components/GridItem.vue';
     import GridLayout from './components/GridLayout.vue';
-    // import ResponsiveGridLayout from './components/ResponsiveGridLayout.vue';
     import TestElement from './components/TestElement.vue';
-    import CustomDragElement from './components/CustomDragElement.vue';
     import {getDocumentDir, setDocumentDir} from "./helpers/DOM";
-    //var eventBus = require('./eventBus');
 
-    let testLayout = [
+    var testLayouts = {
```
Comment (Xhhhhhh) [inline src/App.vue:104]: bug fix?

**Comment by Xhhhhhh** [comment]
1.  check the ci 
2.  the dist dir need push to repository？

File: src/App.vue (hunk 1 above)
Comment (Chellhooo) [reply src/App.vue:104]: no

**Comment by Chellhooo** [comment]
> 1. check the ci
> 2. the dist dir need push to repository？

both yes
all changes based on https://github.com/jbaysolutions/vue-grid-layout/compare/master...vue3-webpack

File: src/App.vue (hunk 1 above)
Comment (Chellhooo) [reply src/App.vue:104]: all changes based on https://github.com/jbaysolutions/vue-grid-layout/compare/master...vue3-webpack

File: src/App.vue [hunk 2]
```diff
@@ -281,9 +296,11 @@
             },
             layoutReadyEvent: function(newLayout){
                 console.log("Ready layout: ", newLayout)
+                this.layout = newLayout
```
Comment (Xhhhhhh) [inline src/App.vue:299]: can pr "support vue3" only for upgrading vue from 2 to 3?  bug fix or implementation change move to another pr.

Comment (Chellhooo) [reply src/App.vue:299]: all changes based on https://github.com/jbaysolutions/vue-grid-layout/compare/master...vue3-webpack

this is what origin author do to support vue3

**Review by Xhhhhhh — APPROVED** [review]
